addon/components/datepicker-support.js

```diff
@@ -52,6 +52,14 @@ export default Ember.Mixin.create({
         if (!self.$().val()) {
           self._applyDateValue(null);
         }
+      }).
+      on('focusout', function() {
+        console.log('focusout');
+        self.sendAction('focusOut');
```

Comment:
Yes, absolutely. Should have checked out the Ember.TextSupport beforehand :wink: